pubspec.yaml

```diff
@@ -101,8 +101,6 @@ dependencies:
   shared_preferences: ^2.2.3
   shared_preference_app_group: ^1.1.1 # iOS Notification Service extension requirement to access shared preferences
   simple_animations: ^5.0.2
-  sqflite_common_ffi: ^2.3.3
```

Comment:
Do you need it as a dev dependency for testing?

Reply:
We don't have unit testing on Misty Breez yet.